src/olympia/addons/tests/test_utils_.py

```diff
@@ -103,7 +104,7 @@ def test_recommended(self, incr_mock):
     def test_recommended_no_results(self, incr_mock):
         self.recommendation_server_mock.return_value = []
         recommendations, outcome, reason = get_addon_recommendations('a@b', True)
-        assert recommendations == TAAR_LITE_FALLBACKS
+        assert recommendations == TAAR_LITE_FALLBACKS[:4]
```

Comment:
This is a bit ambiguous.. I guess you are slicing the array because you expect the sliced item to be the "excluded" addon. Is there a more explicit way of doing that?

Maybe assert that the one you expect not to be there is not in there.

```python
assert "foo" not in recommendations
```

That is easier to understand and a more precise test.

Reply:
I feel like the existing test works better in context, in that "if there's no results, it only takes the first four fallbacks".

I added a comment and I can still add that test, but arbitrarily setting the precedent to test for n-many fallbacks not being in it seems unneccessary?

Reply:
That was from the original product decision. We know the frontend should never display more than 4, and the fewer the API returns, the faster it's going to be (those responses can be quite verbose, so any kilobyte counts).

https://mozilla.github.io/addons-server/topics/api/addons.html#recommendations documents that it will return 4 max.

Reply:
Ok. That's rather unclear then. Maybe a comment in the function linking to the docs would help.